javascripts/projectsService.js

```diff
@@ -50,16 +45,13 @@
     }
 
     // Make sure the names are sorted first. Then return the found index in the passed names
-    return _.filter(
-      _.map(projectNamesSorted, function(entry, key) {
-        if (names.indexOf(String(key)) > -1) {
-          return entry;
-        }
-      }),
-      function(entry) {
-        return entry || false;
+    return  _.filter(_.map(projectNamesSorted, function(entry, key) {
```

Comment:
@savvas94 There are multiple spaces before "_", Also your PR have lots of other errors which are shown in travis logs. please have a look on those.